modules/govuk_jenkins/files/var/lib/jenkins/groovy_scripts/govuk_jenkinslib.groovy

```diff
@@ -13,6 +13,19 @@ def cleanupGit() {
 }
 
 /**
+ * Checkout repo using SSH key
+ */
+def checkoutSSH(String repository, String org = 'alphagov', String url = 'github.com') {
+  checkout([$class: 'GitSCM',
+    branches: scm.branches,
+    userRemoteConfigs: [[
+      credentialsId: 'govuk-ci-ssh-key',
+      url: "git@${url}:${org}/${repository}.git"
```

Comment:
Yep, and that makes sense to me. What I meant is that the way the url is build looks more specific than the name might suggest, e.g. you might run in to difficulties if you tried to use this to checkout something not on a GitHub instance.